statics/js/components/workflow.js

```diff
@@ -99,6 +96,26 @@ Vue.component('workflow-params', {
 
   methods: {
 
+    beautifyDesc: function(out) {
+      var i = 0
+      var href = function() {
+        i++;
+        return '<h1 onclick="$(\'#wf-arrow-' + i + '\').toggleClass(\'down\')" \
```

Comment:
tested, it works fine!
I feel the title looks big like the main one, how about reducing the size to ```<h5>``` or ```<h4>```
just my suggestion ;-)

Reply:
I can reduce a bit the size of all h* tags of markdown conversion